package.json

```diff
@@ -112,7 +112,8 @@
             "box-ui-elements-locale-data": "<rootDir>/i18n/en-US.js",
             "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/conf/jest/fileMock.js",
             "\\.(css|less|scss)$": "<rootDir>/conf/jest/mocks/styleMock.js",
-            "react-virtualized/dist/es": "react-virtualized/dist/commonjs"
+            "react-virtualized/dist/es": "react-virtualized/dist/commonjs",
+            "globals": "<rootDir>/conf/jest/mocks/globalsMock.js"
```

Comment:
Can we define `__VERSIONS__` in `env-setup.js`?

Reply:
Tried this, and it doesn't seem to work. We can't set __VERSIONS__ directly because it doesn't exist globally yet, and setting global.__VERSIONS__ is different from what the plugin does, so we can't read that value in our util method.